File: php/Mahasiswa.php
<?php

class Mahasiswa {

	private $nim = '';
	private $nama = '';
	private $gender = '';
	private $fakultas = '';
	private $prodi = '';

	public function __construct($nim = '', $nama = '', $gender = '', $fakultas = '', $prodi = ''){
		$this->nim = $nim;
		$this->nama = $nama;
		$this->gender = $gender;
		$this->fakultas = $fakultas;
		$this->prodi = $prodi;
	}

	public function set_nim($nim){
		$this->nim = $nim;
	}

	public function set_name($name){
		$this->name = $name;
	}

	public function set_gender($gender){
		$this->gender = $gender;
	}

	public function set_fakultas($fakultas){
		$this->fakultas = $fakultas;
	}

	public function set_prodi($prodi){
		$this->prodi = $prodi;
	}

	public function get_nim(){
		return $this->nim;
	}

	public function get_nama(){
		return $this->nama;
	}

	public function get_gender(){
		return $this->gender;
	}

	public function get_fakultas(){
		return $this->fakultas;
	}

	public function get_prodi(){
		return $this->prodi;
	}

}

?>

File: java/Mahasiswa.java
//deklarasi class Mahasiswa
public class Mahasiswa{

	//deklarasi atribut yang dimiliki mahasiswa, semuanya bertipe string
	private String nim;
	private String nama;
	private String gender;
	private String prodi;
	private String fakultas;

	//default constructor
	public Mahasiswa(){
		this.nim = "";
		this.nama = "";
		this.gender = "";
		this.prodi = "";
		this.fakultas = "";
	}

	//constructor dengan atribut sebagai parameternya
	public Mahasiswa(String nim, String nama, String gender, String prodi, String fakultas){
		this.nim = nim;
		this.nama = nama;
		this.gender = gender;
		this.prodi = prodi;
		this.fakultas = fakultas;
	}

	//setter untuk mengset value pada atribut

	public void set_nim(String nim){
		this.nim = nim;
	}

	public void set_nama(String nama){
		this.nama = nama;
	}

	public void set_gender(String gender){
		this.gender = gender;
	}

	public void set_prodi(String prodi){
		this.prodi = prodi;
	}

	public void set_fakultas(String fakultas){
		this.fakultas = fakultas;
	}

	//getter untuk mengambil value pada atribut

	public String get_nim(){
		return this.nim;
	}

	public String get_nama(){
		return this.nama;
	}

	public String get_gender(){
		return this.gender;
	}

	public String get_prodi(){
		return this.prodi;
	}

	public String get_fakultas(){
		return this.fakultas;
	}
}

